design/ui/pagination/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@espressive/pagination",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "module": "es/index.js",
+  "scripts": {
+    "rollup": "rm -rf dist && rm -rf es && BABEL_ENV=production rollup -c",
+    "rollup:watch": "rm -rf dist && rm -rf es && BABEL_ENV=production rollup -c --watch"
```

Comment:
These commands need to become make commands to make them more portable and easier to add to packages. But... they MIGHT also become something we run at the root of the package if this is a thing we need to do across multiple packages consistently. I still do not know what the DX is going to be for us when working on these. For now, they can stay scoped to the packages alone.